pkg/cmd/clustertask/start_test.go

```diff
@@ -313,6 +313,7 @@ func Test_ClusterTask_Start(t *testing.T) {
 				"-p", "print=boom,boom",
 				"-l", "key=value",
 				"-o", "code-image=output-image",
+				"-w", "name=test,claimName=pvc3",
```

Comment:
Why do we need to add `-w` to every test? Wouldn't it be better to just create a test specifically for workspaces?

Reply:
The TaskSpec of the ClusterTask-1 consists of a workspace and that's why used the `-w` flag so that we don't get the prompt to input the workspace.

Reply:
Can you create new test data instead of adding it? This will help to keep original tests as they were.

Reply:
@danielhelfand anyhow we need to modify the existing tasks because in the existing test data workspace is already defined so for their respective taskruns we need to add `-w` flag but sure I will add one more test data for the workspaces